Formula/hepmc3.rb

```diff
@@ -13,18 +13,17 @@ class Hepmc3 < Formula
 
   bottle do
     root_url "https://ghcr.io/v2/davidchall/hep"
-    sha256 cellar: :any, monterey: "d2273c38945cc73c136c961213f3b710caf1b10293be65fffbafeeb8ffc05f72"
```

Comment:
You shouldn't need to edit these lines. They will be replaced by the merge bot that runs after you add the "pr-pull" label.

Reply:
Yes.